src/main/java/com/crud/tasks/scheduler/EmailScheduler.java

```diff
@@ -11,33 +12,25 @@
 @Component
 public class EmailScheduler {
 
-    private static final String SUBJECT = "Tasks: Once a day email";
+    public static final String SUBJECT = "Tasks: Once a day email";
+    public static final String MESSAGE = "Currently in database you got: ";
 
     @Autowired
     private SimpleEmailService simpleEmailService;
 
     @Autowired
-    private TaskRepository taskRepository;
+    private MailCreatorService mailCreatorService;
 
     @Autowired
     private AdminConfig adminConfig;
 
     @Scheduled(cron = "0 0 10 * * *")
     public void sendInformationEmail(){
 
-        long size = taskRepository.count();
-        String task;
-
-        if (size == 1){
-            task = "task";
-        } else {
-            task = "tasks";
-        }
-
         Mail mail = new Mail(
                 adminConfig.getAdminMail(),
                 SUBJECT,
-                "Currently in database you got: " + size + " " + task + ".",
+                mailCreatorService.buildInformationEmail(MESSAGE),
```

Comment:
jesli zawsze message jest ten sam, to prawdopodobnie lepiej w ogole wyrzucic ten parametr. Message moze byc albo w metodzie buildInformationEmail, albo zahardcodowany w szablonie

Reply:
dalej nierozwiazane